android-plugin/src/main/resources/io/ecocode/xml/rules/ecocode_xml_profile.json

```diff
@@ -1,13 +1,13 @@
 {
```

Comment:
why does this file not moved to the new path ("...../rules/xml/....") ?

Reply:
Path `io.ecocode.rules.`_`language`_ will be moved into : `green-code-initiative:ecoCode/ecocode-rules-specifications`

I don't think it is a good idea to link profil of implemented rules to rule specifications and to put those JSON files into  `green-code-initiative:ecoCode/ecocode-rules-specifications`.
I think it is better to keep profiles directly in plugins.